dicognito/__main__.py

```diff
@@ -14,13 +14,25 @@ def main(args=None):
     if args is None:
         args = sys.argv[1:]
 
-    parser = argparse.ArgumentParser(description='Process some integers.')
-    parser.add_argument('patterns', metavar='pattern', type=str, nargs='+',
-                        help='the files to anonymize (may include wildcards, e.g. *.dcm)')
+    parser = argparse.ArgumentParser(description="Anonymize one or more DICOM files.")
+    parser.add_argument("patterns", metavar="pattern", type=str, nargs="+",
+                        help="the files to anonymize (may include wildcards, e.g. *.dcm)")
+    parser.add_argument("--id-prefix", default="",
```

Comment:
Can we have -p and -s as well?

Reply:
I was leery of putting it in. What if we want `-s` for something even better?
But probably, yeah.